### ISSUE TITLE
"g:startify_session_dir" wrong default value for Vim/Windows

### ISSUE THREAD

**Issue by EtiamNullam** [mhinz/vim-startify#397]
According to `:help g:startify_session_dir` it should default to: `$HOME/vimfiles/session` for Vim/Windows, but it actually is `$HOME/.vim/session`

**Comment by EtiamNullam** [comment]
I was wrong - it was probably some leftover session (SpaceVim maybe?) that kept reapearring.

**Comment by EtiamNullam** [comment]
Actually I'm not sure if `g:startify_session_dir` should be responsible for it but a file `default.vim` gets created at `~/.vim/sessions` whenever I close vim without custom session saved, `:SSave` saves in a proper `~/vimfiles/session`.

**Comment by mhinz** [comment]
Disclaimer first: I don't have a Windows for testing at the moment.

The default path is set only once when Startify gets sourced, so when `:SS` is working as intended, it's a bit weird when `~/.vim/session` gets created anyway. Honestly, I don't think Startify is the issue here.

Do you have any other session plugins? If so, disable them during testing. When you use `:SS`, the internal Vim variable `v:this_session` gets set to that path. Before using `:SS`, and without any other session plugin, `v:this_session` is usually empty.

- Just after starting Vim, what does `:echo v:this_session` print?
- Maybe recursively search your Vim plugin folder for `default.vim`.

**Comment by EtiamNullam** [comment]
- `echo v:this_session` - nothing is printed.
- With empty `vimrc` - no session is created.
- When I load only the startify plugin - **works as expected**.
- With `vimrc` where I only load all my plugins - the issue persists, but **if I do `:SS` before leaving Vim - the above `session` isn't created.**

Weird.

==============

I narrowed it down to `coc.nvim` - it creates that session file even without `startify`, thanks for your time!

**Comment by mhinz** [comment]
Yeah, I'm guessing that CoC creates the `default.vim` session file on its own unless `v:this_session` is set. At least that would explain all 4 points.

Just create another issue with them. It's probably easy to fix. :)
